### ISSUE TITLE
window border visible on kde

### ISSUE THREAD

**Issue by kineticz** [Psykopear/fuzzle#1]
Here is a screenshot:

![afbeelding](https://user-images.githubusercontent.com/995761/75431244-ddb16b00-594c-11ea-9e0a-4305d262c9dc.png)

**Comment by Psykopear** [comment]
Yes, right now this is blocked by https://github.com/xi-editor/druid/issues/427

**Comment by Psykopear** [comment]
I see the size of the window does not look right too, this is probably an issue related to some hardcoded sizes that work differently on different resolutions/dpi

**Comment by Psykopear** [comment]
So, I fixed the issue on Druid and added the options here on fuzzle. If you can confirm me the fix works I'll close the issue.
